bundles/plugins/org.bonitasoft.studio.connectors/src-test/java/org/bonitasoft/studio/connectors/ui/property/section/ConnectorEventFilterTest.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (C) 2015 BonitaSoft S.A.
+ * Bonitasoft, 32 rue Gustave Eiffel - 38000 Grenoble
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2.0 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.bonitasoft.studio.connectors.ui.property.section;
+
+import static org.bonitasoft.studio.model.process.builders.ConnectorBuilder.aConnector;
+import static org.bonitasoft.studio.model.process.builders.TaskBuilder.aTask;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.bonitasoft.engine.bpm.connector.ConnectorEvent;
+import org.eclipse.jface.viewers.Viewer;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ConnectorEventFilterTest {
+
+	@Mock
+	private Viewer viewer;
+
+	private ConnectorEventFilter onFinishFilter;
+
+	@Before
+	public void setup() {
```

Comment:
To have more readable test it's highly recommended to use @Before and @After sparingly.
In your use case you can easily extract the instantiation of your filter in a private method and call it each one of your test.